### PR TITLE
[WIP] Add Drupal 8

### PR DESCRIPTION
**Pull request by fureigh**
DO NOT MERGE.

### Known TODOs:
* [ ] Update README to account for both Drupal 7 and 8.
* [ ] Finish solving deployment issue. (Currently it's possible to get partway through an install but then it fails.)

### CODE REVIEW

File: drupal-7/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+jobs:
+- name: deploy-drupal-7
+  plan:
+  - aggregate: *plan-gets
```
Comment (jmcarp) [inline drupal-7/pipeline.yml]: This reference doesn't have a corresponding anchor, so it's not valid.

File: drupal-7/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - put: deploy-drupal-7
+    params:
+      path: cf-ex-drupal
+      manifest: cf-ex-drupal/manifest.yml
+  on_failure:
+    put: slack
+    params:
+      <<: *slack-params
```
Comment (jmcarp) [inline drupal-7/pipeline.yml]: Same as above--there's no `&slack-params` anchor to reference.

File: drupal-7/pipeline.yml (hunk 1 above)
Comment (fureigh) [reply drupal-7/pipeline.yml]: Thanks! I hadn't meant to include that particular file; I've removed it now and will finish it up in a different branch.

**Comment by pburkholder** [comment]
Closing this since superseded by #12

**Pull request closed without merging**